mappings/net/minecraft/server/world/ServerTickScheduler.mapping

```diff
@@ -1,16 +1,72 @@
 CLASS bbh net/minecraft/server/world/ServerTickScheduler
+	FIELD a filter Ljava/util/function/Predicate;
 	FIELD b idToName Ljava/util/function/Function;
 	FIELD c nameToId Ljava/util/function/Function;
 	FIELD d ticksScheduled Ljava/util/Set;
 	FIELD e ticksScheduledOrdered Ljava/util/TreeSet;
 	FIELD f world Luk;
 	FIELD g ticksCurrent Ljava/util/List;
 	FIELD h tickConsumer Ljava/util/function/Consumer;
+	METHOD <init> (Luk;Ljava/util/function/Predicate;Ljava/util/function/Function;Ljava/util/function/Function;Ljava/util/function/Consumer;)V
+		ARG 1 world
+		ARG 2 filter
+		ARG 3 idToName
+		ARG 4 nameToId
+		ARG 5 tickConsumer
 	METHOD a tick ()V
+		ARG 1 tasksCount
+		ARG 2 iter
+		ARG 3 tick
+		ARG 6 report
+		ARG 7 el
 	METHOD a toTag (Lbag;)Lhw;
+		ARG 1 pos
+		ARG 2 ticks
+		ARG 3 worldTime
+		ARG 5 tags
+		ARG 8 nbt
+	METHOD a (Lcap;Let;)V
+		ARG 1 box
+		ARG 2 offset
+		ARG 3 ticks
+		ARG 6 pos
+	METHOD a getScheduledTicksInBox (Lcap;Z)Ljava/util/List;
+		ARG 1 box
+		ARG 2 remove
+		ARG 3 ticks
+		ARG 5 iter
+		ARG 6 tick
+		ARG 7 pos
 	METHOD a isScheduled (Let;Ljava/lang/Object;)Z
 		ARG 1 pos
+		ARG 2 data
 	METHOD a schedule (Let;Ljava/lang/Object;ILbbl;)V
+		ARG 1 pos
+		ARG 2 data
+		ARG 3 delay
+		ARG 4 priority
 	METHOD a fromTag (Lhw;)V
+		ARG 1 tags
+		ARG 3 nbt
+		ARG 4 data
+	METHOD a getScheduledTicksInChunk (ZLbag;)Ljava/util/List;
+		ARG 1 remove
+		ARG 2 pos
+		ARG 3 x
+		ARG 4 x1
+		ARG 5 z
+		ARG 6 z1
+	METHOD b isTicking (Let;Ljava/lang/Object;)Z
+		ARG 1 pos
+		ARG 2 data
 	METHOD b scheduleForced (Let;Ljava/lang/Object;ILbbl;)V
-	METHOD c scheduleInternal (Let;Ljava/lang/Object;ILbbl;)V
+		ARG 1 pos
+		ARG 2 data
+		ARG 3 delay
+		ARG 4 priority
+	METHOD c (Let;Ljava/lang/Object;ILbbl;)V
```

Comment:
What about just `schedule`?

Reply:
`schedule` is used elsewhere in the tick scheduler,
So I though it would be better have something different. This method is what actually does the work of creating a `ScheduledTick` instance and adding it to the queue.

Reply:
> What about just `schedule`?

A method with that name/signature already exists, I'll do addScheduledTick